### PR TITLE
[Task] YM-21449: CocoaPods sample application

### PR DESCRIPTION
**Pull request by rgcriado**
## Purpose
Create a sample application of the library for CocoaPods integration.

### CODE REVIEW

File: Samples/CocoaPods/Sample/Podfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Podfile:3]:
```suggestion
```

File: Samples/CocoaPods/Sample/Sources/Sample/Delegates/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+    
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Sources/Sample/Delegates/AppDelegate.swift:12]:
```suggestion
```

File: Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+import UIKit
+
+@available(iOS 13.0, *)
+class SceneDelegate: UIResponder, UIWindowSceneDelegate {
+    var window: UIWindow?
+
+    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
+        // Use this method to optionally configure and attach the UIWindow `window` to the provided UIWindowScene `scene`.
+        // If using a storyboard, the `window` property will automatically be initialized and attached to the scene.
+        // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift:14]:
```suggestion
```

File: Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        guard let _ = (scene as? UIWindowScene) else { return }
+    }
+
+    func sceneDidDisconnect(_ scene: UIScene) {
+        // Called as the scene is being released by the system.
+        // This occurs shortly after the scene enters the background, or when its session is discarded.
+        // Release any resources associated with this scene that can be re-created the next time the scene connects.
+        // The scene may re-connect later, as its session was not necessarily discarded (see `application:didDiscardSceneSessions` instead).
+    }
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift:23]:
```suggestion
    func sceneDidDisconnect(_ scene: UIScene) {}
```

File: Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+    func sceneDidBecomeActive(_ scene: UIScene) {
+        // Called when the scene has moved from an inactive state to an active state.
+        // Use this method to restart any tasks that were paused (or not yet started) when the scene was inactive.
+    }
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift:28]:
```suggestion
    func sceneDidBecomeActive(_ scene: UIScene) {}
```

File: Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+    func sceneWillResignActive(_ scene: UIScene) {
+        // Called when the scene will move from an active state to an inactive state.
+        // This may occur due to temporary interruptions (ex. an incoming phone call).
+    }
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift:33]:
```suggestion
    func sceneWillResignActive(_ scene: UIScene) {}
```

File: Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+    func sceneWillEnterForeground(_ scene: UIScene) {
+        // Called as the scene transitions from the background to the foreground.
+        // Use this method to undo the changes made on entering the background.
+    }
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift:38]:
```suggestion
    func sceneWillEnterForeground(_ scene: UIScene) {}
```

File: Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+    func sceneDidEnterBackground(_ scene: UIScene) {
+        // Called as the scene transitions from the foreground to the background.
+        // Use this method to save data, release shared resources, and store enough scene-specific state information
+        // to restore the scene back to its current state.
+    }
```
Comment (jowie) [inline Samples/CocoaPods/Sample/Sources/Sample/Delegates/SceneDelegate.swift:44]:
```suggestion
    func sceneDidEnterBackground(_ scene: UIScene) {}
```

**Review by jowie — APPROVED** [review]

**Comment by rgcriado** [comment]
@jowie I've removed all the commented lines as you suggested 👍

**Review by mkotsiandris — APPROVED** [review]

**Pull request merged**